routes/api/index.js

```diff
@@ -13,4 +13,6 @@ router.use((err, req, res, next) => {
   return next(err);
 });
 
+router.use('/auth/', require('../api/authRoutes/authRoutes'));
```

Comment:
`'../api/authRoutes/authRoutes'` is not cool enough, please make it `'../api/authRoutes'`

Reply:
i'll change that